src/index.html

```diff
@@ -5,9 +5,53 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&family=Poppins:wght@400;500;600;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27"
+        class="card__img"
+      >
+      <p class="card__text card__text--y">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+      <p class="card__code">
+        Product code: 195434
+      </p>
+
+      <div class="stars stars--y">
+        <ul class="stars__wrap">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+        <span class="stars__number">Reviews: 5</span>
+      </div>
+
+      <div class="card__price card__price--y">
+        <p class="card__price-title">Price:</p>
+        <p class="card__price-text">$2,199</p>
+      </div>
+      <a
+        href="#"
+        class="btn"
+        data-qa="hover"
+      >
+      
+        Buy
+
+      </a>
```

Comment:
No need add empty line inside tags
```suggestion
      >
        Buy
      </a>
```